templates/_helpers.tpl

```diff
@@ -31,6 +31,55 @@ Expand the name of the chart.
 {{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{/*
+Compute if the csi driver is enabled.
+*/}}
+{{- define "vault.csiEnabled" -}}
+{{- $_ := set . "csiEnabled" (or
+  (and (ne (.Values.csi.enabled | toString) "-") (eq (.Values.csi.enabled | toString) "true"))
```

Comment:
Is the `(ne (.Values.csi.enabled | toString) "-")` part redundant? I'm reading it as `(A != B) && (A == C)`, which seems equivalent to `A == C` as a test, but I could be reading it wrong.

Reply:
I think you're right. I'll simplify this.